doc/nrf/libraries/networking/wifi_prov_core.rst

```diff
@@ -1,26 +1,25 @@
-.. _wifi_prov_readme:
+.. _lib_wifi_prov_core:
```

Comment:
I suggest to keep both the labels to resolve doc build issue.

Reply:
oh I just pushed the changes to replace this even in older changelogs.